docs/_pages/httpresponsemessages.md

```diff
@@ -0,0 +1,28 @@
+---
+title: HttpResponseMessages
```

Comment:
❌ This doesn't work, unless you also update https://github.com/fluentassertions/fluentassertions/blob/master/docs/_data/navigation.yml